cypress/integration/features/teams/createEditDeleteTeam.feature

```diff
@@ -0,0 +1,41 @@
+Feature: CRUD operation on Teams
+
+  As a teacher I want to create/edit/delete the team so that I can manage the team.
+
+Scenario: Creating a new team by Teacher
+  Given I am logged in as a 'teacher' at 'brb'
+  When I go to teams overview
+  When I click on add team button
+  Then I see new team creation page
+  When I enter in the title 'this is a team'
+  When I enter in the description 'this is team description'
+  Then I see team chat activation check box is visible
+  Then team video activation check box is visible
+  When I select the team colour to red
+  Then I click on create button
+  When I go to teams overview
+  Then I see team titled 'this is a team' is visible
```

Comment:
Typo error:
`Then I see team titled 'this is a team' is visible` ---> `Then I see team title 'this is a team' is visible`

Reply:
done!